docs/wrappers/peer_version_preamble.md

```diff
@@ -8,7 +8,7 @@ information. For example:
  peer:
    Version: 2.1.0
    Commit SHA: b78d79b
-   Go version: go1.13.8
+   Go version: go1.14.1
```

Comment:
This is an example; it doesn't matter what it is.

Reply:
Agreed it doesn't technically matter, but having it accurate relative to the Version field above can only reduce user confusion.